### PR TITLE
740 fix

### PR DESCRIPTION
**Pull request by andipavllo**
@argelius when you have time please take a look at this. I decresed the transaction delay from 0.2s to 0.15s as it seems to be similar to the default ios/android behavior.

### CODE REVIEW

File: css-components/components-src/stylus/components/list.styl [hunk 1]
```diff
@@ -256,12 +256,9 @@ list__item()
 .list__item--tappable
   transition background-color 0.2s linear
 
-.list__item--tappable:active
+.list__item--tappable-active
```
Comment (argelius) [inline css-components/components-src/stylus/components/list.styl]: It looks like the modifier is called "tappable-active". Can you change to `.list__item--tappable--active`?

File: core/elements/ons-list-item.es6 [hunk 2]
```diff
@@ -55,6 +61,14 @@ limitations under the License.
     _shouldLockOnDrag() {
       return this.hasAttribute('lock-on-drag');
     }
+
+    _onTouch() {
```
Comment (argelius) [inline core/elements/ons-list-item.es6]: Write test for this method.

File: core/elements/ons-list-item.es6 [hunk 3]
```diff
@@ -55,6 +61,14 @@ limitations under the License.
...
+        this.className = this.className + ' ' + 'list__item--tappable-active';
+    }
+
+    _onRelease() {
```
Comment (argelius) [inline core/elements/ons-list-item.es6]: Write test for this method.

**Comment by andipavllo** [comment]
@argelius should be ok now

**Comment by argelius** [comment]
@andipavllo This breaks the behavior if you're only use the CSS components. Please don't remove the `:active` pseudoclass from the CSS Components.

**Comment by argelius** [comment]
@andipavllo I will merge this today since were actually removing the `tappable` modifier.

**Comment by andipavllo** [comment]
@argelius oh I see, do I still need to fix the CSS? I had no time to do it in the last few days.

**Comment by argelius** [comment]
@andipavllo I have added the "tappable" attribute as we discussed in the meeting. This attribute will work nice on iOS. I restored the "tappable" modifier so there is still an option if you only use the CSS.

**Pull request merged**